Configurations/15-ios.conf

```diff
@@ -34,6 +34,29 @@ my %targets = (
         inherit_from     => [ "ios-common" ],
         CC               => "xcrun -sdk iphonesimulator cc",
     },
+    "iossimulator-arm64-xcrun" => {
+        inherit_from     => [ "ios-common" ],
+        CC               => "xcrun -sdk iphonesimulator cc",
+        cflags           => add("-arch arm64 -mios-simulator-version-min=7.0.0"),
+        bn_ops           => "SIXTY_FOUR_BIT_LONG",
```

Comment:
Same here
```suggestion
        bn_ops           => "SIXTY_FOUR_BIT_LONG RC4_CHAR",
```

Reply:
I want to understand when to use `RC4_CHAR` and `RC4_INT`. I've created this issue: https://github.com/openssl/openssl/issues/21358

Reply:
So it seems like the advice is not to use these definitions.